components/ModalRoadMap.vue

```diff
@@ -0,0 +1,107 @@
+<template>
```

Comment:
from mobile, keeping the line vertical creates this horizontal scroll, perhaps it is better if from mobile we remove the vertical line and make the content appear in the centre, every time you click a dot it turns green and the content changes.
<img width="193" alt="Screenshot 2024-05-27 alle 17 10 00" src="https://github.com/logix-software/frontend-assessment/assets/12210564/bb31978b-4dbd-48e5-b8ff-57a684dd9c61">

Reply:
on desktop the vertical line is not always aligned with the circle
<img width="1018" alt="Screenshot 2024-05-27 alle 19 52 04" src="https://github.com/logix-software/frontend-assessment/assets/12210564/e25c553d-7b9e-4871-ab95-85db66eccde1">